src/components/Trade/TradeInput.tsx

```diff
@@ -557,8 +557,7 @@ export const TradeInput = () => {
         },
       ]
     if (isBelowMinSellAmount) {
-      return activeSwapperName !== undefined &&
```

Comment:
Ditto, does the Gas oracle change means we should never show the "Or Invalid Trade Pair" state for 1inch? At first glance this seems unrelated

Reply:
Reason for these two changes is that the one inch quote wasn't handling sell amounts below min gracefully. It now is so we can display in the same manner as the other swappers.